web/lens/src/styles.css

```
@@ -129,9 +129,13 @@
     to { opacity: 1; }
   }
 
+  /* Fluid, not fixed: the drawer hosts real charts, and a slab pinned at 720px
+     spent the same width on a 1280px laptop and a 2560px desk. It keeps 720 as
+     its floor (the width every drawer was designed against) and grows with the
+     viewport up to a ceiling, so the page behind it is still visibly the page. */
   .lens-drawer {
     @apply lens-flex lens-h-[100dvh] lens-w-full lens-flex-col lens-bg-card lens-text-text lens-shadow-2xl;
-    max-width: min(720px, calc(100vw - 24px));
+    max-width: min(clamp(720px, 56vw, 1040px), calc(100vw - 24px));
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)